src/wrapper/clap-juce-wrapper.cpp

```diff
@@ -992,6 +997,7 @@ class ClapJuceWrapper : public clap::helpers::Plugin<
             return editor->isResizable();
         return true;
     }
+
     bool guiAdjustSize(uint32_t *w, uint32_t *h) noexcept override
```

Comment:
I feel like we should probably add a comment for what this method is intended to do, since its purpose might not be immediately obvious.